app/utils/answersHelpers/AnswersHelper.scala

```diff
@@ -57,6 +57,27 @@ class AnswersHelper(userAnswers: UserAnswers)(implicit messages: Messages) exten
         )
     }
 
+  def getAnswerAndBuildRowWithRemove[T](
```

Comment:
Could this perhaps include the .getOrElse logic as well?

Reply:
yeah maybe, to reduce the duplication in the individual helpers?
 let me look into it.

Reply:
yeah, it might just look a little big